### PR TITLE
Feat : Add grid

### PR DESCRIPTION
**Pull request by OmKakatkar**
- Adds 4 types of grids
  - 2 column grid
  - 3 column grid
  - sidebar-content grid
  - 2 side bars with content in between type grid
- [Code Preview](https://codesandbox.io/s/github/OmKakatkar/Ani-Design/tree/feature_grid).
- Live Preview Available in the comments by Netlify bot .

### CODE REVIEW

**Review by Logan1x — APPROVED** [review]

**Review by Parul1999 — APPROVED** [review]
Looks good to merge

File: grid.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            <div class="container grid grid-col3">
+                <div class="image box-center">
+                    <img src="./assets/card-image.jpg" alt="" />
+                </div>
+                <div class="image box-center">
+                    <img src="./assets/card-image.jpg" alt="" />
+                </div>
+                <div class="image box-center">
+                    <img src="./assets/card-image.jpg" alt="" />
+                </div>
```
Comment (romabulani) [inline grid.html:43]: You should always give value to alt attribute

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

File: grid.html (hunk 1 above)
Comment (OmKakatkar) [reply grid.html:43]: Yes, will keep that in mind while working on the documentation site. Currently they all have empty `alt` attributes

**Pull request merged**